server/utils/operatorUtils.js

```diff
@@ -16,6 +16,34 @@ const normalizeMaturity = maturity => {
   return validMaturityStrings[0];
 };
 
+const getExampleYAML = (kind, operator) => {
+  const examples = _.get(operator, 'metadata.annotations.alm-examples');
+  if (!examples) {
+    return null;
+  }
+
+  try {
+    const yamlExamples = JSON.parse(examples);
+    return _.find(yamlExamples, { kind });
+  } catch (e) {
+    console.error(e);
+  }
+  return null;
+};
+
+const normalizeCRD = (crd, operator) => ({
```

Comment:
are these fields always there? Or can there be a case where crd has no attribute name? Would it be safer to do _.get() and have it default to N/A or something?